digicampipe/tests/test_lid_ccd.py

```diff
@@ -1,6 +1,13 @@
 from digicampipe.image.sky_image import *
 import os
 
+from pkg_resources import resource_filename
```

Comment:
Ah maybe I should explain what I did here. So before I made this change to the tests. We got this error on the continous integration machine:
FileNotFoundError: https://travis-ci.org/calispac/digicampipe/builds/316067756#L956

The reason is that this relative path does not work, when the test is not executed from the /tests folder. And it usually is not. 

To find test resources on the customers system, after the package was installed, there is the module called `pkg_resources`.  This combined with `glob` allows us to find all the fits files. 

For the 2nd part ("'tests/resources/stars_on_lid/*.fits'") I should have used `os.path.join` and write it like:

    os.path.join('tests', 'resources', 'stars_on_lid', '*.fits')

So the path is platform independent. Would be a pity to be the reason why this package does not run on Windows10.

But I was lazy :-( ... sorry.

I also pulled the path finding and list generation out of the test function. This is because I like the test function to contain basically only two things: 
    calling the function/object under test
    asserting the result to be correct.

Generation of "difficult" input arguments, is nothing I want to test. I am sure I can make this list of file paths.
Also I want to tell this to the reader: this test is about calling the `LidCCDObservation` constructor and not about making a list of strings. I do not want to distract the reader from the actual thing under test ... but maybe .. this is not so important here...